Bounding_volumes/doc/Bounding_volumes/CGAL/Approximate_min_ellipsoid_d.h

```diff
@@ -137,6 +137,8 @@ for display in Maplesoft's Maple.
 
 \cgalExample{Approximate_min_ellipsoid_d/ellipsoid_for_maple.cpp} 
 
+\note This class requires the \ref thirdpartyEigen library.
```

Comment:
Should not we also say that, in case the user does not use our CMake scripts, then the macro `CGAL_EIGEN3_ENABLED` must be defined. Maybe that could be added to [the page about Eigen](https://cgal.geometryfactory.com/~sgiraudot/Solver_interface_remove_diagonalize_traits/Manual/installation.html#thirdpartyEigen).

Reply:
I don't think so, the code should explicitly instantiate the code with the traits class using Eigen. The macro definition is here to filter out overloads where the default of the traits is used.

Reply:
OK, I understand. Than maybe our CMake scripts should not add `-DCGAL_EIGEN3_ENABLED` everywhere, to test that it compiles correctly.

Reply:
Here there is a problem: there is no possible way to instantiate the code with a traits class or another as it is all internal. So here it all depends on the macro `CGAL_EIGEN3_ENABLED`: if it is not used, then the fallback mode is used and with this PR, the user will get a deprecation warning.

The clean thing I can do is to remove the fallback and only keep Eigen, but that will probably mean error messages for people who do not include Eigen in their CMakeLists.